cloud_governance/common/aws/cost_explorer/cost_explorer_operations.py

```diff
@@ -0,0 +1,32 @@
+import json
+from datetime import datetime, timedelta
+
+import boto3
+
+
+class CostExplorerOperations:
+    """
+    This class extracts the price data from AWS Cost Explorer
+    """
+
+    def __init__(self):
+        self.cost_explorer_client = boto3.client('ce')
+
+    def get_cost_usage(self, tag: str, metrics_type: str):
+        """
+        This method extracts the IAM price by Tag provided
```

Comment:
This method extracts the IAM price by Tag provided - is it the right term  IAM price ?